tests/tests_suite_test.go

```diff
@@ -65,7 +65,7 @@ func TestTests(t *testing.T) {
 
 	enableJunit := os.Getenv("JUNIT")
 	if enableJunit == "true" {
-		junitReporter := reporters.NewJUnitReporter("junit.xml")
+		junitReporter := reporters.NewJUnitReporter(homeHome + "junit.xml")
```

Comment:
Shouldn't this be `filepath.Join(homeHome, "junit.xml")`? It's not obvious to me that `homeHome` already ends with the right path separator.